packages/debugger/webpack/webpack.config.js

```diff
@@ -1,4 +1,4 @@
-const merge = require("webpack-merge");
+const {merge} = require("webpack-merge");
```

Comment:
Incredibly nitpicky style comment: Should be `{ merge }` instead of `{merge}`. :P  (We're supposed to have `prettier` to handle such things, but it's broken at the moment... @cds-amal is working on getting it working again, though.)

Reply:
I shall fix.